Persephone/Controllers/WindowController.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  WindowController.swift
+//  Persephone
+//
+//  Created by Daniel Barber on 2019/1/11.
+//  Copyright © 2019 Dan Barber. All rights reserved.
+//
+
+import Cocoa
+import mpdclient
+
+class WindowController: NSWindowController {
+  enum TransportAction: Int {
+    case prevTrack = 0
+    case playPause = 1
+    case stop = 2
+    case nextTrack = 3
+  }
```

Comment:
In the event you add more items to the enum, to make it simpler, you could omit all the individual values (and even the `Int` declaration if that's not important):

```
enum TransportAction {
  case prevTrack, playPause, stop, nextTrack
}
```